DataFormats/Calibration/include/DataFormatsCalibration/MeanVertexObject.h

```diff
@@ -15,29 +15,38 @@
 #include <array>
 #include "Rtypes.h"
 
+#include "Framework/Logger.h"
+#include "ReconstructionDataFormats/Vertex.h"
+
 namespace o2
 {
 namespace dataformats
 {
-class MeanVertexObject
+class MeanVertexObject : public VertexBase
```

Comment:
we need also getters for mean vertex at given Z, i.e.

```
VertexBase getMeanVertex(float z) 
{
  VertexBase v = *this;
  float dz = getZ() - z;
  v.setXYZ(v.getX()+mSlopeX*dz, v.getY()+mSlopeY*dz, z);
}

const VertexBase& getMeanVertex() const {
  return (const VertexBase&)(*this);
}
```

Reply:
Done, using the functions already in the class.